tools/mmgotool/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/mattermost/mattermost/tools/mmgotool
```

Comment:
Is it worth maintaining a submodule for this tool, or could we just merge this with the server's?

Reply:
If tools are also being consumed by other projects, I'd incline to have them separately within `/tools` directory at top level. But I'm okay with general consensus. One of my motivation is to have tools collected within the same directory so that it would be much easier to discover.

Reply:
I love the idea of boosting discoverability. And putting it outside the `server/` makes the `go.mod` decision very straightforward.